### ISSUE TITLE
Consider double line breaks as new paragraph when pasting

### ISSUE THREAD

**Issue by mtias** [WordPress/gutenberg#12638]
Let's consider treating two contiguous line breaks as a new paragraph when pasting.

**Comment by ellatrix** [comment]
In my testing, this is already the case. @mtias could you confirm that it works properly for you?

Try to paste this (`test<br><br>test`):

test<br><br>test

**Comment by mapk** [comment]
I just tested this out with a copy/paste from Text Edit. It didn't work in relation to pasting directly into the Title. It just pasted everything into the Title. When I placed my cursor where I wanted the break to happen and clicked `Enter` it just moved my cursor to a new block, but didn't move my text to the new block. I had select the text and remove it manually and then paste it into the new block.

![pasting](https://user-images.githubusercontent.com/617986/78915916-86211780-7a41-11ea-8b11-b852a3af253b.gif)
